.github/workflows/update_dependencies.yml

```diff
@@ -29,19 +29,17 @@ jobs:
       - name: Update dependencies
         run: lake update
 
-      - name: Generate branch name
-        id: generate_branch_name
+      - name: Generate PR title
         run: |
-          timestamp=$(date -u +"%Y-%m-%d-%H-%M")
-          echo "branch_name=update-dependencies-$timestamp" >> "$GITHUB_ENV"
+          echo "timestamp=$(date -u +"%Y-%m-%d-%H-%M")" >> "$GITHUB_ENV"
           echo "pr_title=chore: update Mathlib dependencies $(date -u +"%Y-%m-%d")" >> "$GITHUB_ENV"
 
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v6
         with:
           token: "${{ secrets.NIGHTLY_TESTING }}"
           commit-message: "chore: update Mathlib dependencies ${{ env.timestamp }}"
```

Comment:
It looks like `env.timestamp` was always blank because `timestamp` wasn't being saved to `$GITHUB_ENV`.